provider/hybrid.go

```diff
@@ -85,6 +82,9 @@ func (dp dockerHybridProvider) Configure(
 
 	// For the most part delegate Configure handling to the bridged provider.
 	resp, err := dp.bridgedProvider.Configure(ctx, request)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I've overlooked potential panic in reps.SupportsPreview below if err is non-nil. This now prevents such a panic.